app/src/main/java/org/greenstand/android/TreeTracker/signup/CredentialEntryView.kt

```diff
@@ -175,6 +185,53 @@ fun <T : Credential> CredentialButton(
     }
 }
 
+@Composable
+fun ExistingUserDialog(
+    viewModel: SignupViewModel,
+    navController: NavHostController,
+    state: SignUpState
+) {
+    AlertDialog(
+        onDismissRequest = {
+            viewModel.closeExistingUserDialog()
+        },
+        title = {
+            Text(text = stringResource(R.string.user_exists_header))
+        },
+        text = {
+            Text(
+                text = stringResource(R.string.user_exists_message),
+                color = Color.Green
+            )
+        },
+        buttons = {
+            state.existingUser?.let {
+                UserButton(
+                    user = it,
+                    isSelected = false,
+                    AppButtonColors.Default,
+                    Green
+                ) {
+                    navController.navigate(NavRoute.Dashboard.route) {
+                        popUpTo(NavRoute.Language.route) { inclusive = true }
+                        launchSingleTop = true
+                    }
```

Comment:
It looks like if we click on an existing user, we then instantly log them out. We should log them in, and go to the wallet screen.